### PR TITLE
Scotto37

### PR DESCRIPTION
**Pull request by joe-scotto**
Add Scotto37 keyboard along with small updates to my corne keymap.

Add Scotto37 keyboard along with small updates to my corne keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/scotto37/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include QMK_KEYBOARD_H
```
Comment (spidey3) [inline keyboards/scotto37/keymaps/default/keymap.c:1]: This file needs a copyright notice.

**Review by spidey3 — CHANGES_REQUESTED** [review]
LGTM; just needs copyright notice on one file.

File: keyboards/scotto37/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# scotto37
+
+![scotto37](imgur.com image replace me!)
+
+_A short description of the keyboard/project_
```
Comment (zvecr) [inline keyboards/scotto37/readme.md:5]: Needs filling out

File: keyboards/scotto37/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: _The PCBs, controllers supported_
+-   Hardware Availability: _Links to where you can find this hardware_
```
Comment (zvecr) [inline keyboards/scotto37/readme.md:9]: Needs filling out

File: keyboards/scotto37/keymaps/default/config.h [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Define default keymap options
+#define USE_SERIAL
```
Comment (zvecr) [inline keyboards/scotto37/keymaps/default/config.h:21]:
```suggestion
```

File: keyboards/scotto37/config.h [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Joe Scotto (@joe-scotto)
```
Comment (zvecr) [inline keyboards/scotto37/config.h:1]: This file should be removed

File: keyboards/crkbd/keymaps/joe_scotto/config.h [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+/* disable print */
+// #define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+#define USE_SERIAL
```
Comment (zvecr) [inline keyboards/crkbd/keymaps/joe_scotto/config.h:64]:
```suggestion
```

File: keyboards/scotto37/keymaps/default/config.h [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
```
Comment (zvecr) [inline keyboards/scotto37/keymaps/default/config.h:16]:
```suggestion
// Copyright 2022 Joe Scotto (@joe-scotto)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/scotto37/keymaps/default/keymap.c (hunk 1 above)
Comment (zvecr) [reply keyboards/scotto37/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2022 Joe Scotto (@joe-scotto)
// SPDX-License-Identifier: GPL-2.0-or-later
#include QMK_KEYBOARD_H
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**